pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -44,7 +44,7 @@ while (( "$n" < "$nParams" )); do
 
     case "$p" in
         -[cSEM] | -MM) dontLink=1 ;;
-        -cc1) cc1=1 ;;
+        -cc1 | -fc1) cc1=1 ;;
```

Comment:
No space

Reply:
I put the spaces here to be consistent in style with the other case statements. Plus, if I remove them, `shfmt` adds them back :\